src/main/java/wooteco/subway/ui/ExceptionController.java

```diff
@@ -1,21 +1,22 @@
 package wooteco.subway.ui;
 
-import org.springframework.dao.EmptyResultDataAccessException;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
+import wooteco.subway.exception.BusinessException;
 
 @RestControllerAdvice
 public class ExceptionController {
 
-    @ExceptionHandler({IllegalArgumentException.class, EmptyResultDataAccessException.class})
-    public ResponseEntity<String> handelIllegalArgument(Exception e) {
+    @ExceptionHandler(BusinessException.class)
+    public ResponseEntity<String> handelBusinessException(Exception e) {
```

Comment:
```suggestion
    @ExceptionHandler(BusinessException.class)
    public ResponseEntity<String> handelBusinessException(BusinessException e) {
```

커스텀 예외까지 잘 구현해주셨어요 👍 
커스텀 예외를 사용하면 앱에서 직접 정의한 예외 메세지만 요청 클라이언트에게 노출될 수 있겠죠. 잘 생각해서 구현해주셨습니다.

여기는 사소하지만 예외의 범위를 조금 더 줄일 수 있을 것 같네요 😄

Reply:
칭찬 감사합니다! BusinessException으로 수정 완료 했습니다 :)